src/WingmanDataTrail/MetricVizPanel/MetricVizPanel.tsx

```diff
@@ -201,7 +198,7 @@ export class MetricVizPanel extends SceneObjectBase<MetricVizPanelState> {
     metricName,
     matchers,
     prometheusFunction,
-    groupByLabel,
+    //groupByLabel,
```

Comment:
```suggestion
```

Reply:
its an unused variable after the changes but i didn't want to completely remove it in case we want to revert later but if i leave it uncommented eslint complains

Reply:
With all of that in mind, I still suggest removing this line before merging